integration/experiment/machine.py

```diff
@@ -113,6 +116,13 @@ def _query(self):
         for sn in signal_names:
             self.signals[sn] = util.geopmread('{} board 0'.format(sn))
 
+        conditional_signal_names = ['GPU_CORE_FREQUENCY_STEP']
+        for sn in signal_names:
```

Comment:
Just a note/reminder to address the issue we noted in #2747

Reply:
👍 Addressed in #2747